{{cookiecutter.project_slug}}/_/deployment/python/deployment/kubernetes/application/overlays/production/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../base
```

Comment:
Maybe let's scale to 2 replicas for the prod env?

Reply:
Added that 👍